SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -171,54 +171,54 @@ def processInteractionEvents(self, callerInteractor, eventId, viewWidget):
       try:
         xy = callerInteractor.GetEventPosition()
         import vtkSegmentationCorePython as vtkSegmentationCore
-        masterImageData = self.getClippedMasterImageData()
-        ijk = self.xyToIjk(xy, viewWidget, masterImageData)
+        sourceImageData = self.getclippedSourceImageData()
+        ijk = self.xyToIjk(xy, viewWidget, sourceImageData)
         self.floodFillFromPoint(ijk)
       except IndexError:
-        logging.error('apply: Failed to threshold master volume!')
+        logging.error('apply: Failed to threshold source volume!')
       finally:
         qt.QApplication.restoreOverrideCursor()
       abortEvent = True
 
     return abortEvent
 
   def floodFillFromPoint(self, ijk):
-    """Fills the segment taking based on the current master volume.
-    Input IJK position is voxel coordinates of master volume.
+    """Fills the segment taking based on the current source volume.
+    Input IJK position is voxel coordinates of source volume.
     """
     self.scriptedEffect.saveStateForUndo()
 
-    # Get master volume image data
+    # Get source volume image data
     import vtkSegmentationCorePython as vtkSegmentationCore
-    masterImageData = self.getClippedMasterImageData()
+    sourceImageData = self.getclippedSourceImageData()
     selectedSegmentLabelmap = self.scriptedEffect.selectedSegmentLabelmap()
 
     # Get modifier labelmap
     modifierLabelmap = self.scriptedEffect.defaultModifierLabelmap()
 
-    pixelValue = masterImageData.GetScalarComponentAsFloat(ijk[0], ijk[1], ijk[2], 0)
+    pixelValue = sourceImageData.GetScalarComponentAsFloat(ijk[0], ijk[1], ijk[2], 0)
 
     useSegmentationAsStencil = False
 
     # Perform thresholding
     floodFillingFilter = vtk.vtkImageThresholdConnectivity()
-    floodFillingFilter.SetInputData(masterImageData)
+    floodFillingFilter.SetInputData(sourceImageData)
     seedPoints = vtk.vtkPoints()
-    origin = masterImageData.GetOrigin()
-    spacing = masterImageData.GetSpacing()
+    origin = sourceImageData.GetOrigin()
+    spacing = sourceImageData.GetSpacing()
     seedPoints.InsertNextPoint(origin[0]+ijk[0]*spacing[0], origin[1]+ijk[1]*spacing[1], origin[2]+ijk[2]*spacing[2])
     floodFillingFilter.SetSeedPoints(seedPoints)
 
     maskImageData = vtkSegmentationCore.vtkOrientedImageData()
-    intensityBasedMasking = self.scriptedEffect.parameterSetNode().GetMasterVolumeIntensityMask()
+    intensityBasedMasking = self.scriptedEffect.parameterSetNode().GetSourceVolumeIntensityMask()
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases. -- I will not add this comment to any more places...